modules/claims_api/lib/bgs_service/local_bgs.rb

```diff
@@ -429,5 +417,31 @@ def jrn
         jrn_obj_id: Settings.bgs.application
       }
     end
+
+    private
+
+    def transform_keys(hash_or_array)
+      transformer = lambda do |object|
+        case object
+        when Hash
+          object.deep_transform_keys! { |k| k.underscore.to_sym }
+        when Array
+          object.map { |item| transformer.call(item) }
+        else
+          object
+        end
+      end
+
+      transformer.call(hash_or_array)
+    end
+
```

Comment:
This is a refactor of `parsed_response`. I made this method and `transform_keys` private since they are helper methods. Rather than test them directly, I think we are covered by existing tests plus the new test for `findPOAs` (in which the response is an array).